src/main/java/nl/tudelft/scrumbledore/game/MultiPlayerGameFactory.java

```diff
@@ -0,0 +1,16 @@
+package nl.tudelft.scrumbledore.game;
+
+/**
+ * Creates an instance of a Multi Player Game.
+ * 
+ * @author Niels Warnars
+ */
+public class MultiPlayerGameFactory extends GameFactory {
+
+  
+  @Override
+  Game createGame() {
```

Comment:
@Override should stay present.
Reference: https://stackoverflow.com/questions/1005898/java-should-i-add-an-override-annotation-when-implementing-abstract-methods
<organisation